core/servicedata/servicedata.go

```diff
@@ -0,0 +1,23 @@
+package servicedata
+
+import (
+	"context"
+	"fmt"
+)
+
+type Repository interface {
+	CreateServiceDataKey(ctx context.Context, serviceDataKey ServiceDataKey) (ServiceDataKey, error)
+}
+
+type ServiceDataKey struct {
```

Comment:
How about calling it as 
```suggestion
type Key struct {
```
only?